src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -2,17 +2,19 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import { PostgreSQLManagementClient } from '@azure/arm-postgresql';
-import { ServerForCreate } from '@azure/arm-postgresql/src/models';
+import { PostgreSQLManagementClient, PostgreSQLManagementModels as SingleModels } from "@azure/arm-postgresql";
+import { PostgreSQLManagementClient as PostgreSQLFlexibleManagementClient, PostgreSQLManagementModels as FlexibleModels } from "@azure/arm-postgresql-flexible";
 import { Progress } from 'vscode';
 import { AzureWizardExecuteStep, callWithMaskHandling, createAzureClient, LocationListStep } from 'vscode-azureextensionui';
 import { ext } from '../../../../extensionVariables';
 import { localize } from '../../../../utils/localize';
 import { nonNullProp } from '../../../../utils/nonNull';
+import { AbstractServerCreate, PostgresServerType } from '../../../abstract/models';
 import { IPostgresServerWizardContext } from '../IPostgresServerWizardContext';
 
 export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresServerWizardContext> {
     public priority: number = 150;
+    public defaultVersion: string = "11";
```

Comment:
Any specific reason this default changed from 10 to 11? If we're going to update the default, I'd prefer that as a separate PR instead of hidden in this larger refactoring. I'm not super versed on postgres versions, so I'd wanna know what the portal uses as a default, what the support level is for 10 vs 11, etc.

Reply:
The portal default is 11 for single server and 12 for flexible server. 

Single server supports 9.6-11
Flexible server supports 11-13

10 is no longer supported in flexible server, but still in single server (but no longer the default as it was when this was originally written).

I can make it a separate PR but would have to replace this with a switch to default to 10 for single and 11 for flexible

Reply:
Matching the portal would be best, aka 11 for single and 12 for flexible. I think I'm okay including this in the current PR.